djstripe/webhook.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+"""
+Utils related to processing or registering for webhooks
+"""
+from collections import defaultdict
+
+__all__ = ['handler', 'handler_all', 'call_handlers', ]
+
+
+# A model registers itself here if it wants to be in the list of processing
```

Comment:
Please transfer as much of these line comments into the module docstring as you can.